### PR TITLE
Rely on CSV for fetching ocs version info

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar <shmohan@redhat.com>

### CODE REVIEW

**Comment by shylesh** [comment]
image

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -940,17 +941,15 @@ def get_ocs_build_number():
         str: build number for ocs operator version
 
     """
-    from ocs_ci.ocs.resources.catalog_source import CatalogSource
+    from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
```
Comment (clacroix12) [inline ocs_ci/utility/utils.py]: Is there a reason we are importing here as opposed to at the top of the file with all other imports? If so please add a comment explaining why we are importing here.

Comment (shylesh) [reply ocs_ci/utility/utils.py]: Sorry I forgot to put it in comments, there was circular dependency hence I imported it here. I will update the comment

Comment (shylesh) [reply ocs_ci/utility/utils.py]: done

Comment (shylesh) [reply ocs_ci/utility/utils.py]: done

**Comment by shylesh** [comment]
image

**Comment by shylesh** [comment]
Verified with live deployment. 
Version appears as expected in subject and ocs operator

**Pull request merged**